Additional sql user (#181)
nais/doc: docs/persistence/postgres.md

The backups can be found in the [Google Cloud SQL instance](https://cloud.google.com/sql) dashboard.

## Additional user(s) database(s)

You can add users to your database by setting database configuration option: `.spec.gcp.sqlInstances[].databases[].users`. 
Names added must match regex: `^[_a-zA-Z][_a-zA-Z0-9]+$`. Secrets is generated and mounted for each user.
With `.spec.gcp.sqlInstances[].databases[].envVarPrefix` set to `DB` and additional username to `_user2` you will get environment variables in format `DB_USER2_MYDB_USERNAME` etc. Details about environment variables is specified her: [`configuration`](../persistence/postgres.md#configuration)

!!! info

> Note that if you have deployed your application with an additional user, and then change name or remove the user from configuration, you have to manually delete the google-sql-*MYAPP*-*USER*:
> ```bash
> $ kubectl delete secret google-sql-<MYAPP>-<USER>
> ```

Reviewer: ```suggestion
    If you've deployed your application with an [additional user](#additional-users-databases), and then change name or remove the user from configuration, you need to _manually_ delete the `google-sql-<MYAPP>-<USER>` secret:
    ```bash
    $ kubectl delete secret google-sql-<MYAPP>-<USER>
    ```
```
Indent correctly for `!!! info` panel, and some of my personal opinion on the language =)

Reviewer: Two minor tidbits, but one broken `!!! info` panel I suggest you fix =)
